scripts/fix-dependencies.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * This script finds all the `package.json` files under `packages` related
+ * to the hint project. Then it updates any reference to any of these
+ * packages if needed.
+ *
+ * This should be run before and after a release to make sure everything is
+ * OK.
+ */
+
+const fs = require('fs');
+const path = require('path');
+const globby = require('globby');
+
+/** All the `package.json` properties that need to be updated. */
+const propertiesToUpdate = [
+    'dependencies',
+    'devDependencies',
+    'optionalDependencies',
+    'peerDependencies'
+];
+
+/**
+ * Finds and loads all the `package.json` files returning 2 maps:
+ *
+ * * `packages`: The content of each `package.json` indexed by `name`.
+ * * `paths`: The routes to each `package.json`. We can predict the
+ *   path because some packages don't follow the convention (e.g.:
+ *   `extension-browser`).
+ *
+ */
+const initialize = () => {
+    const packagesFiles = globby.sync([
+        '**/*/package.json',
+        '!**/*/fixtures'
```

Comment:
Should this just be a shallow match under `packages/*/package.json` to also avoid `node_modules` and other places we might have a `package.json` that doesn't relate to one of our packages?

Reply:
That's why I have `gitignore: true` so it doesn't match anything under `node_modules`